src/morse/index.js

```diff
@@ -58,6 +58,33 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+    if((typeof text ) != "string"){
+    throw "Please provide a morse string";
+  }
+  text = text.replace(/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g, '');
```

Comment:
What does this do?

This is the type of thing you'd want to put a comment ahead to explain.